### ISSUE TITLE
Question re: max

### ISSUE THREAD

**Issue by andyhasit** [n3-charts/line-chart#455]
When display type is 'dot', the highest dot gets cut in half by the top of the graph (Just like dot series in http://n3-charts.github.io/line-chart/#/examples) which I do not want to happen.

The only way round that which I have found is to manually set the "max" to a few ticks more than the highest value in the data set, like so:

```
var highestValue = determineHighestValue(dataSet);
$scope.myOptions.axes.y.max = highestValue + 3;
```

  - Is there a smarter way of doing this?
  - If not, how about we allow passing a function like `function(lowestValue, highestValue)` to min and max?
  - The max and min aren't actually documented on http://n3-charts.github.io/line-chart/#/docs - should I raise a separate issue for that or even raise a PR?

Thanks :-)

**Comment by chaosmail** [comment]
Duplicate of #474

**Comment by chaosmail** [comment]
Hi @andyhasit 
yeah your workaround is the only solution for now. Sure, feel free to submit a PR for the docs.

You can pass the function to `min` and `max` options, however i think we wont support one single option for both because this would be redundant.
Best,
Christoph

**Comment by chaosmail** [comment]
This has been fixed in #476, please see the issue for the solution
